packages/app/src/components/collapsible/collapsible-section.tsx

```diff
@@ -86,20 +73,23 @@ export const CollapsibleSection = ({
 
 const StyledAnchor = styled(Anchor)(
   css({
-    color: 'gray2',
+    color: colors.gray2,
     px: 3,
     py: 1,
     width: 0,
     textDecoration: 'none',
     position: 'absolute',
     right: '100%',
     '&:hover, &:focus': {
-      color: 'blue',
+      color: colors.blue1,
     },
   })
 );
 
-const Summary = styled.button(
+interface SummaryPropsType {
```

Comment:
Please refrain from "Hungarian notation"-like suffices.
```suggestion
interface SummaryProps {
```